dvc/api.py

```diff
@@ -7,31 +7,38 @@
 
 
 class UrlNotDvcRepoError(DvcException):
-    """Thrown if given URL is not a DVC repository.
-
-    Args:
-        url (str): URL to the repository
-    """
+    """Thrown if the given URL is not a DVC repository."""
 
     def __init__(self, url):
         super().__init__("'{}' is not a DVC repository.".format(url))
 
 
 def get_url(path, repo=None, rev=None, remote=None):
-    """Returns URL to the storage location of a data artifact tracked
-    by DVC, specified by its path in a repo.
+    """
+    Returns the URL to the storage location of a data file or directory tracked
+    in a DVC repo. Its formed by reading the remote configuration, and the
+    DVC-file where the given path is an output. For Git repos, HEAD is used
```

Comment:
This thing of how it's formed blurs things more than explains unless you already very familiar with dvc. I would just say it gives you an url to a copy of file in a remote cache.

Reply:
True. I just removed that one sentence now (and completed the default arg value info, see 7f083979).